### PR TITLE
acre_api_fnc_setCustomSignalFunc - reset fix

### PR DESCRIPTION
**Pull request by Dorbedo**
**When merged this pull request will:**
- fixes the possibility to reset the function to default behavior
- changes the reset value from an emtpy array (nil) into an emtpy code({})

### CODE REVIEW

File: addons/api/fnc_setCustomSignalFunc.sqf [hunk 1]
```diff
@@ -14,14 +14,17 @@
...
  * [mySignalFunc] call acre_api_fnc_setCustomSignalFunc
+ * [{}] call acre_api_fnc_setCustomSignalFunc
  *
  */
 #include "script_component.hpp"
 
-params ["_code"];
-
-if (!(_code isEqualType {})) exitWith {
+If !(_this params [["_code",{},[{}]]]) exitWith {
```
Comment (jonpas) [inline addons/api/fnc_setCustomSignalFunc.sqf]: Lower-case if.

File: addons/api/fnc_setCustomSignalFunc.sqf [hunk 2]
```diff
@@ -14,14 +14,17 @@
...
     ERROR("acre_api_fnc_setCustomSignalFunc called with invalid argument.");
 };
 
-EGVAR(sys_signal,customSignalFunc) = _code;
+If (_code isEqualTo {}) then {
```
Comment (jonpas) [inline addons/api/fnc_setCustomSignalFunc.sqf]: Lower-case if.

**Review by jonpas — CHANGES_REQUESTED** [review]

**Review by jonpas — DISMISSED** [review]

**Comment by Sniperhid** [comment]
What is the advantage of this?

`[] call acre_api_fnc_setCustomSignalFunc;` - Presently works fine to reset the custom signal function.

**Comment by Dorbedo** [comment]
Did you try it in sheduled environment?

**Review by Sniperhid — APPROVED** [review]

**Comment by Sniperhid** [comment]
Good point! This does indeed fix it in that case.

**Pull request merged**